CMakeLists.txt

```diff
@@ -249,6 +249,32 @@ if(ENABLE_LIBXC)
   endif()
 endif()
 
+if(DEFINED DeePMD_DIR)
```

Comment:
Please add this update to https://github.com/deepmodeling/abacus-develop/blob/develop/docs/advanced/install.md